spec/src/main/asciidoc/configsources.asciidoc

```diff
@@ -35,8 +35,6 @@ It can also be used to implement a drop-in configuration approach.
 Simply create a jar containing a `ConfigSource` with a higher ordinal and override configuration values in it.
 If the jar is present on the classpath then it will override configuration values from <<ConfigSource,ConfigSources>> with lower `ordinal` values.
 
-The ordinal for property file based <<ConfigSource,ConfigSources>> can be configured using the key `config_ordinal` inside the property file.
```

Comment:
I'd say we should make clear that the config_ordinal mechanism is guaranteed to work for built-in ConfigSources. And of course we can recommend that other ConfigSources use the same trick. But at the end only the getOrdinal() method counts, regardless how this value got evaluated or defined in the custom ConfigSource. 

We also cannot test every custom ConfigSource in our TCK, so any specification in this area is moot anyway, isn't?